docs/adns9800.md

```diff
@@ -15,4 +15,5 @@ firmware = (
 ```
 
 ## Constructor parameters
-ADNS9800(cs=*cs_pin*, sclk=*clock_pin*, miso=*miso_pin*, mosi=*mosi_pin*, invert_x=*False*, invert_y=*False*)
\ No newline at end of file
+
```

Comment:
Blank line after headings are generally inconsistent. Is there a preferred style?

Reply:
In this case I was just making this match the ptBR version of the same file, since IMO the blank line looks better. Looks like at there is some consensus on this:
https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022